### DISCUSSION TITLE
Tooling Talks Episode 0 - Welcome to Tooling Talks

### DISCUSSION THREAD

**Discussion by giscus** [ckipp01/tooling-talks#57] [Episode Discussion]
# Tooling Talks Episode 0 - Welcome to Tooling Talks

I'm incredibly excited to be starting this journey with you on
  Tooling Talks. Every month I'll be having a conversation with someone working
  in Tooling. We'll dive into tools, talking about the future of tooling, and
  hopefully provide some insight into how to get started in tooling if you new.
  Thanks for joining along.

https://www.tooling-talks.com/episode-0

**Comment by ckipp01** [comment]
Hello world :) Just seeing if the giscuss integration works.

**Reply by ckipp01** [reply]
And does a response work the other way around from GitHub?
